service/model/environment.go

```diff
@@ -119,7 +110,7 @@ func (e *Environment) Update(name, rawURL *string) error {
 	}
 
 	if rawURL != nil {
-		if err := e.SetURL(*rawURL); err != nil {
+		if err := e.SetServiceURL(*rawURL); err != nil {
```

Comment:
> (optional) And actually, it might be even more clean to have the following signature:
> 
>     * `Update(u EnvironmentUpdate) (*Environment, error)`

Not sure if I get the point. What is the benefit of returning a pointer to the Environment in the case of a pointer receiver? Is it just for better readability?

```
env, err := env.Update(...)
```